src/ui/pages/SwapForm/SwapForm.tsx

```diff
@@ -346,10 +346,10 @@ export function SwapFormComponent() {
           chainId: txToCheck.chainId,
           from: address,
           to: txToCheck.to,
-          value: txToCheck.value,
+          value: 'value' in txToCheck ? txToCheck.value : null,
           data: txToCheck.data,
           gas,
-          nonce,
+          nonce: Number(nonce),
```

Comment:
txNonce type shows as unknown few lines before. Let's defined type for it too?

Reply:
Why we are using number here anyway and a bit later we use `nonce: nonce != null ? Number(nonce) : undefined,`?
Shouldn't nonce exist all the time here and not as 0?